src/scss/includes/components/mainActionButton.scss

```diff
@@ -1,6 +1,6 @@
 .mainActionButton {
   cursor: pointer;
-  padding: 8px;
+  padding: 8px 0px;
```

Comment:
I would keep a small value instead of 0px, to prevent two consecutive long strings appear completely glued.
`4px` looks ok to keep "Supermarché" full, but maybe its safer to use `2px` to have some margin for the different fonts depending on platforms.
BTW, this is a good case to use "official" spacing values :) See `spacing.scss` for ref.

Reply:
:+1: 
I'm not sure if `border-radius: 8px;` right bellow shouldn't be treated a spacing and left as it is?